Add unique topics handling and fix rate calc

savex opened the pull request:
Bool flag unique-topics makes producers and
    consumers generate topics using counters in
    "topic-<num>" format.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/redpanda-data/client-swarm?pullRequest=18) <br/>All committers have signed the CLA.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/redpanda-data/client-swarm?pullRequest=18) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/redpanda-data/client-swarm?pullRequest=18) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/redpanda-data/client-swarm?pullRequest=18) it.</sub>

StephanDollberg commented on src/main.rs:
```diff
@@ -435,17 +447,19 @@ enum Commands {
     Producers {
         #[clap(short, long)]
-        topic: String,
+        topic_prefix: String,
```
Changing `--topic` to `--topic-prefix` arg probalby breaks a bunch of invocations in various places so it's not worth changing. 

Instead I'd either just keep `--topic` and only use `--unique_topics` or just add `--topic-prefix` and then require either `--topic` or `--topic-prefix`.

savex replied:
Yep, I thought to update it to better reflect the content. But fixing all those in main repo is a pain. Let me update this.

StephanDollberg reviewed (commented):
It would be great to have two separate commits for the new arg and the rate fix. Not immediately clear what is wrong with the later.

savex commented:
> It would be great to have two separate commits for the new arg and the rate fix. Not immediately clear what is wrong with the later.

When the `total_time` has very small value `as_secs()` will produce `0` seconds. And after that, comes division by zero in the next formula.

savex committed:
Fix division by zero

    start_time.elapsed will produce zero on rare occasions. Add the
    check and do not do calculation.

savex committed:
Fix duplicated short option chars

    messages_per_second is colliding with messages and timeout_ms
    colliding with topic

savex committed:
Add unique topics handling

    Bool flag unique-topics makes producers and
    consumers generate topics using counters in
    "topic-<num>" format
